stdlib/3/collections/__init__.pyi

```diff
@@ -62,6 +62,7 @@ else:
 _UserDictT = TypeVar('_UserDictT', bound=UserDict)
 
 class UserDict(MutableMapping[_KT, _VT]):
+    data = ...  # type: Mapping[_KT, _VT]
```

Comment:
Don't you want to type this as `Dict[_KT, _VT]`? The docs say it's a "real dict". (OTOH the Python 2 stub, `UserDict.pyi`, also uses `Mapping`.)

Reply:
That's the main reason why I kept it as a mapping. Happy to change to Dict.